acmwebsite/templates/index.xhtml

```diff
@@ -9,11 +9,9 @@
   <body py:block="body" py:strip="True">
     <div class="row">
       <div class="col-md-7">
-        <div class="home-banner">
-          <img class="home-banner-img" src="${tg.url('/img/banner.jpg')}" />
-          <small>
-          Image above: ACM Members and other clubs at a Mozilla Campus Clubs training event
-          </small>
+        <div py:if="banner is not None" class="home-banner">
+          <img class="home-banner-img" src="${banner.photo.url}" />
+          <small py:if="banner.description" py:content="banner.description"></small>
```

Comment:
As @jackrosenthal, mentioned in an earlier review, this should be markdownified. (See `templates/meeting_list.xhtml:5` for an example.